k8soperators/k8soperators.github.io
# code block format for ch3

# What
code block format for ch3
# Verification
- checkout this branch
- make sure you have node 20 on your pc
- run `npm run build` and `npm run preview`  
in your browser on http://localhost:4321/ you should see the below codeblock behaviour
![cb2](https://github.com/k8soperators/k8soperators.github.io/assets/16667688/3d4a34dc-d6c1-465c-8ba1-e91e305f0b34)
